src/screens/Onboarding/Layout.tsx

```diff
@@ -116,8 +116,7 @@ export function Layout({children}: React.PropsWithChildren<{}>) {
         contentContainerStyle={{borderWidth: 0}}
         // @ts-ignore web only --prf
         dataSet={{'stable-gutters': 1}}>
-        <View
-          style={[a.flex_row, a.justify_center, gtMobile ? a.px_5xl : a.px_xl]}>
+        <View style={[a.flex_row, a.justify_center]}>
```

Comment:
Need to remove the padding from the layout so that the horizontal `FlatList`s appear correctly.